src/resources/marketplace.js

```diff
@@ -173,10 +174,8 @@ class Marketplace {
    * @return {Promise<{timestamp, transactionReceipt}>}
    */
   async acceptOffer(id, data, confirmationCallback) {
```

Comment:
I think all of the `data` params should have default values so that they can be omitted.

E.g. `acceptOffer(id, data = {}, confirmationCallback)`